integration-tests/integration_test.go

```diff
@@ -834,6 +843,90 @@ func (s *ProcessListeningOnPortTestSuite) TestProcessListeningOnPort() {
 	assert.True(s.T(), hasOpenAndClose(endpoints9091), "Did not capture open and close events for port 9091")
 }
 
+func (s *SymbolicLinkProcessTestSuite) SetupSuite() {
+
+	s.metrics = map[string]float64{}
+	s.executor = NewExecutor()
+	s.StartContainerStats()
+	s.collector = NewCollectorManager(s.executor, s.T().Name())
+
+	s.collector.Env["COLLECTOR_CONFIG"] = `{"logLevel":"debug","turnOffScrape":false,"scrapeInterval":2}`
+	s.collector.Env["ROX_PROCESSES_LISTENING_ON_PORT"] = "true"
+
+	err := s.collector.Setup()
+	s.Require().NoError(err)
+
+	err = s.collector.Launch()
+	s.Require().NoError(err)
+	time.Sleep(30 * time.Second)
+
+	processImage := qaImage("quay.io/rhacs-eng/qa", "collector-processes-listening-on-ports")
+
+	containerID, err := s.launchContainer("process-ports", "-v", "/tmp:/tmp", processImage)
+
+	s.Require().NoError(err)
+	s.serverContainer = containerShortID(containerID)
+
+	actionFile := "/tmp/action_file_ln.txt"
+
+	_, err = s.collector.executor.Exec("sh", "-c", "rm "+actionFile+" || true")
```

Comment:
[nit] I think the `sh -c` part is redundant based on the way the executor works (in that gcloud ssh and ssh will run commands via sh -c themselves)

Reply:
Does the action file even exist at this point?

Reply:
I think `sh -c` is needed when running the tests locally. Also when running locally there is a chance that the action file exists. We should be defensive and not make assumptions.

Reply:
Then this indicates a shortcoming in the local executor. We have executor functions for copying files, perhaps we could add one for writing files as well? We are making an assumption here that the executor is not running the command in a shell, which is false in two thirds of the scenarios.

Reply:
We shouldn't ignore this, but happy for it to be completed in a follow up